data/campaigns/Winds_of_Fate/scenarios/04_Islands.cfg

```diff
@@ -0,0 +1,612 @@
+        recruit=Merman Fighter, Merman Hunter, Mermaid Initiate
+        {INCOME 16 32 48}
+        team_name=merfolk
+        user_team_name= _ "Merfolk"
+        [leader]
+            id=Nemas
+            name= _ "Nemas"
+            type=Merman Warrior
+            facing=sw
+        [/leader]
+    [/side]
+
+    [side]
+        side=3
+        controller=ai
+        defeat_condition=no_units_left
+        recruit=Naga Dirkfang, Naga Fighter
+        gold=22
+        {INCOME 22 22 22}
+        team_name=nagas
+        user_team_name= _ "Nagas"
+        [leader]
+            id=Shussek
+            name= _ "Shussek"
+            type=Naga Warrior
+            facing=sw
+        [/leader]
+        [ai]
+            aggression=0.6
+            passive_leader=yes # so he doesn't get himself killed
+        [/ai]
+    [/side]
+
+    [side]
+        side=4
+        controller=ai
+        defeat_condition=no_units_left
+        recruit=Merman Fighter, Merman Hunter, Mermaid Initiate
+        gold=32
+        {INCOME 16 32 48}
+        team_name=merfolk
+        user_team_name= _ "Merfolk"
+        [leader]
+            id=Poseira
+            name= _ "Poseira"
+            type=Merman Spearman
+            facing=se
+        [/leader]
+        [ai]
+            aggression=0.6
+            passive_leader=yes # so he doesn't get himself killed
+        [/ai]
+    [/side]
+
+    # wmllint: validate-on
+
+    [event]
+        name=prestart
+        {MODIFY_UNIT side=1 facing se}
+        [hide_unit]
+            side=1
+        [/hide_unit]
+        # Clashers will be returned in a future scenario when the drakes can forge weapons & armor
+        [store_unit]
+            [filter]
+                side=1
+                type_adv_tree=Drake Clasher
+            [/filter]
+            variable=clashers
+            kill=yes
+        [/store_unit]
+        [disallow_recruit]
+            side=1
+            type=Drake Clasher
+        [/disallow_recruit]
+        {VARIABLE drakes_killed_nagas no}
+        [objectives]
+            side=1
+            [objective]
+                description= _ "Be holding all villages at the third day's end. (Turn 18)"
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Fail to hold all villages at the third day's end. (Turn 18)"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Reshan"
+                condition=lose
+            [/objective]
+            [gold_carryover]
+                bonus=yes
+                carryover_percentage=0
+            [/gold_carryover]
+            [note]
+                description= _ "If we kill none of the naga, we may convince them to ally with us by moving Reshan adjacent to their leader."
```

Comment:
Here "may" is quite ambiguous, I'd expect clearer language in the notes. I wasn't sure if the alliance would always be accepted, or if there was a chance that I'd have ended turn with Reshan next to a still-hostile leader.

Reply:
Also Sea Serpent is not naga but if we kill him we can no longer ally with them. This could be a bit unclear as well.